docs/_extra/api-reference/schemas/group.yaml

```diff
@@ -2,43 +2,41 @@ Group:
   type: object
   required:
     - id
+    - groupid
     - links
     - name
     - organization
     - public
     - scoped
-    - groupid
     - type
   properties:
     id:
       type: string
+    groupid:
+      type: string
+      description: >
+        Authority-unique identifier that may be set for groups that
+        are owned by a third-party authority. This field is currently present
+        but unused for first-party-authority groups.
+    name:
+      type: string
     links:
       type: object
       properties:
         html:
           type: string
           format: uri
           description: URL to the group's main (activity) page
-    name:
-      type: string
     organization:
       description: "**EXPANDABLE** The organization to which this group belongs. May be `null`, even if expanded, as not all groups belong to an organization"
       oneOf:
-          - type: null
```

Comment:
This is the one _material_ change I had to make to both this JSON schema and that of `organization`. OpenAPI doesn't allow JSON Schema body properties to have a `type` of `null` (it won't validate). There is a new property, `nullable` (boolean) that can be used, but is not compatible with swagger 2.0. I'll continue to sort this out as I go.